deploy/helm/sumologic/conf/events/events.conf

```diff
@@ -0,0 +1,29 @@
+<source>
+  @type prometheus
+  metrics_path /metrics
+  port 24231
+</source>
+<source>
+  @type prometheus_output_monitor
+</source>
+<source>
+  @type events
+  deploy_namespace $NAMESPACE
```

Comment:
this is causing issue. I tried changing this line to `deploy_namespace {{ .Release.namespace }}` expecting helm would render it from the `tpl` function [here](https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/118/files#diff-f6b30aed1be608276081c7b71149b30fR11), but it's just empty string if I do that

Reply:
oops that fixed it! 😂 thanks Frank